packages/frontend/components/ToTheTopButton/ToTheTopButton.module.scss

```diff
@@ -0,0 +1,33 @@
+@import 'styles/variables';
+
+.container {
+  display: grid;
+  justify-content: center;
+}
+
+.button {
+  display: grid;
+  grid-template-columns: auto auto;
+  align-items: center;
+
+  color: $white;
+  background: #373737;
+  height: 2.4em;
+  width: 8.4em;
+  border-radius: 10px;
+
+  font-family: Roboto;
```

Comment:
use the default font that was specified with the latest merge. I accidentally used roboto in the designs i think.
```suggestion
```